metaflow/plugins/argo/generate_input_paths.py

```diff
@@ -0,0 +1,23 @@
+import sys
+from hashlib import md5
+
+
+def generate_input_paths(step_name, timestamp, input_paths, max_splits):
```

Comment:
`input_paths` have been `flow/run/step/task` previously but it seems that now `input_paths` are `run/step/task`?